Buildings/package.mo

```diff
@@ -307,7 +319,7 @@ have been <b style=\"color:blue\">improved</b> in a
     <td valign=\"top\">Changed the default heating maximum airflow setpoint to 30% of the zone nominal airflow.<br/>
                        This is for
                        <a href=\"https://github.com/lbl-srg/modelica-buildings/issues/2172\">issue 2172</a>.
-</tr>
+</tr>    
```

Comment:
remove the empty space at the end

Reply:
Addressed (no idea how it got there in the first place)